Add how-to guide: applying customizations to variant target languages (#328)
DeepLcom/api-docs: docs/learning-how-tos/examples-and-guides/customizations-for-variants.mdx

---
title: "How to Apply Customizations to Language Variants"
description: "Learn how to apply glossaries and style rules when translating into language variants like FR-CA or EN-GB."
public: true
---

DeepL supports regional language variants such as `PT-BR` vs `PT-PT`, or `FR-CA` vs `FR-FR`. When applying customizations like glossaries and style rules, customizations must be created using the root language code, but can be applied to any variant of that language.

**This guide shows you:**

> - Why customizations must be created with root codes (e.g. `PT`, `FR`) and not variant codes

Reviewer: ```suggestion
- How to create customizations using root language codes (e.g. `PT`, `FR`)
```

> - How to pass a root-code glossary or style rule when translating into a variant target

Reviewer: ```suggestion
- How to pass a customization when translating into a variant target (e.g. `pt-BR`, `fr-CA`)
```

- A practical example of using variant-specific glossaries to enforce locale-appropriate terminology

## Creating a customization for a language 

DeepL distinguishes between root language codes (e.g. `pt`) and language variant codes (e.g. `pt-BR`). Customizations like glossaries and style rules must be created with the root code. Attempting to create one with a variant code will fail.

| **Use this** | **Not this** |
|:---|:---|
| `zh` | `zh-Hant`, `zh-Hans` |
| `pt` | `pt-BR`, `pt-PT` |
| `fr` | `fr-CA`, `fr-CH` |
| `de` | `de-CH` |
| `it` | `it-CH` |
| `es` | `es-ES`, `es-419` |

> See [supported glossary languages](/docs/getting-started/supported-languages) for the full list.

Reviewer: ```suggestion
See [supported languages](/docs/getting-started/supported-languages) for the full list.
```
